lib/extensions/ar_adapter/ar_dba.rb

```diff
@@ -869,4 +869,8 @@ def vacuum_analyze_table(table)
   def vacuum_full_analyze_table(table)
     execute("VACUUM FULL ANALYZE #{quote_table_name(table)}")
   end
+
+  def vacuum_full_table(table)
```

Comment:
I think we should probably go with the `vacuum_full_analyze_table` here. No reason not to really.